src/include/sof/audio/format.h

```diff
@@ -10,6 +10,8 @@
 #ifndef AUDIO_FORMAT_H
 #define AUDIO_FORMAT_H
 
```

Comment:
@tlauda Please explain in the commit message why is this change needed. I assume that this include is needed to fix some compilation error, right?

Reply:
@dbaluta Done.